load_test.py

```diff
@@ -0,0 +1,96 @@
+import time
+from sdcm.tester import ClusterTester
+from sdcm.db_stats import PrometheusDBStats
+
+
+class LoadTest(ClusterTester):
+    """
+    Test Scylla load with cassandra-stress
+    """
+
+    def run_load_and_check_for_admission_control(self, prepare_base_cmd, stress_base_cmd, job_num=8):  # pylint: disable=invalid-name
+        # pylint: disable=too-many-locals
```

Comment:
why is pylint complain for too-many-locals? fix it.

Reply:
not anymore, as this function was divided into few other smaller functions